e2e/js_image_oci/WORKSPACE

```diff
@@ -39,9 +39,9 @@ npm_repositories()
 
 http_archive(
     name = "contrib_rules_oci",
-    sha256 = "85197657c25b5769664c6413988b415a14d35b6dab1754b00d8ae60019936777",
-    strip_prefix = "rules_oci-0.2.1",
-    url = "https://github.com/bazel-contrib/rules_oci/archive/v0.2.1.tar.gz",
+    sha256 = "d11b6e5646c4897fde03e7704ab97305691d871fb8d9e09443384122582e70ff",
+    strip_prefix = "rules_oci-0.3.0",
```

Comment:
nit: 0.3.1 is out

Reply:
Real task here is to wire up the `--experimental_credential_helper`

Reply:
I did a bit of work on that I know how to wire it though i don't have a working prototype yet.